src/core_plugins/input_control_vis/public/components/vis/range_control.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import InputRange from 'react-input-range';
+import { FormRow } from './form_row';
+
+export class RangeControl extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      sliderValue: this.props.control.value
+    };
+
+    this.handleOnChange = this.handleOnChange.bind(this);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState({ sliderValue: nextProps.control.value });
+  }
+
+  handleOnChange(control, value) {
+    this.props.stageFilter(this.props.controlIndex, value, this.props.control.filterManager.createFilter(value));
+  }
+
+  render() {
+    return (
+      <FormRow
+        label={this.props.control.label}
+      >
+        <div className="inputRangeContainer">
+          <InputRange
```

Comment:
Please label this element, using [`aria-labelledby`](https://github.com/davidchin/react-input-range#arialabelledby-string) on the `InputRange` and set this to the `id` of the label in `FormRow` (I guess that needs some modifications.